index.js

```diff
@@ -29,22 +31,25 @@ const sendUAEvent = (event, search) => {
   visitor.event(event, search).send();
 };
 
+// collecting data from website
 const refreshAPI = async () => {
+  // resets all fuzzySet
   guideReset();
   itemReset();
   charReset();
   shopReset();
   stampReset();
-  
+
   const { allItems, allCharacters, allGuides, allShops, allStamps, root } = (await axios.get(API_URL)).data;
   currentData = { allItems, allCharacters, allGuides, allShops, allStamps };
-
+  // console.log("current data ->", currentData);
   root.weapons.forEach(({ id, name }) => {
     weaponHash[id] = name;
   });
-
+  console.log(aliasingLightning(allItems[0]))
   allItems.forEach(item => {
-    addItem(item);
+    // change all the lightning to thunder
+    addItem(aliasingLightning(item));
```

Comment:
when you're adding up item the aliasingLightning function will be invoked with item as argument,
the aliasing function will return a copy object of item that has 'lightning' word in its factors property and replace the 'lightning' word to 'thunder'